docs/Killswitches.md

```diff
@@ -45,15 +63,19 @@ Plugins may use the killswitch system simply by hosting their own version of the
 using `killswitch.get_kill_switches(target='https://example.com/myplugin_killswitches.json')`. The returned object can
 be used to query the kill switch set, see the docstrings for more information on specifying versions.
 
+The version of the JSON file will be automatically upgraded if possible by the code KillSwitch code.
+
 ## Currently supported killswitch strings
 
+ <!-- TODO: update this with new behaviour for various fields -->
 The current recognised (to EDMC and its internal plugins) killswitch strings are as follows:
-| Kill Switch                                           | Description                                                                                  |
-| :---------------------------------------------------  | :------------------------------------------------------------------------------------------- |
-| `plugins.eddn.send`                                   | Disables all use of the send method on EDDN (effectively disables EDDN updates)              |
-| `plugins.(eddn\|inara\|edsm\|eddb).journal`           | Disables all journal processing for EDDN/EDSM/INARA                                          |
-| `plugins.(edsm\|inara).worker`                        | Disables the EDSM/INARA worker thread (effectively disables updates) (does not close thread) |
-| `plugins.(eddn\|inara\|edsm).journal.event.$eventname`| Specific events to disable processing for                                                    |
+| Kill Switch                                            | Description                                                                                                                                    |
+| :----------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------- |
+| `plugins.eddn.send`                                    | Disables all use of the send method on EDDN (effectively disables EDDN updates)                                                                |
+| `plugins.(eddn\|inara\|edsm\|eddb).journal`            | Disables all journal processing for EDDN/EDSM/INARA                                                                                            |
+| `plugins.(edsm\|inara).worker`                         | Disables the EDSM/INARA worker thread (effectively disables updates) (does not close thread)                                                   |
+| `plugins.(edsm\|inara).worker.$eventname`              | Disables the EDSM/INARA worker for the given eventname, OR if delete_fields exists, removes the fields from the event                          |
```

Comment:
> Also, you're using | just to show here that it could apply to any of the core plugins, rather than as a possible syntax in the actual killswitch right?
Yeah, I have a bad habit of sticking regex where it doesnt belong

>PyCharm's MK rendering barfs on this, still interpreting the | as a table column separator.
sounds like a bug report to me :D but yes Ill fix it